egs/librimix/ConvTasNet/eval.py

```diff
@@ -154,9 +153,12 @@ def wer_from_hsdi(hits, substitutions, deletions, insertions):
 
     @staticmethod
     def hsdi(truth, hypothesis):
-        keep = ["hits", "substitutions", "deletions", "insertions"]
+        keep = {"hits": 0, "substitutions": 0, "deletions": 0, "insertions": 0}
         out = compute_measures(truth=truth, hypothesis=hypothesis).items()
-        return {k: v for k, v in out if k in keep}
```

Comment:
Didn't this work? What was the error message?

Reply:
Yes it did ! My bad. The problems comes from `Counter()`.  That removes keys if they are non positive. I am gonna find a workaround ;) 
```python
>>> a = Counter({'menu': 20, 'good': 15, 'happy': 10, 'bar': 0})
>>> b = Counter()
>>> a+b
Counter({'menu': 20, 'good': 15, 'happy': 10})
```

Reply:
Done